RELEASE/scripts/autoscend/auto_familiar.ash

```diff
@@ -63,6 +62,16 @@ boolean isAttackFamiliar(familiar fam)
 	{
 		return true;
 	}
+	if(fam == $familiar[Mini-Adventurer])
+	{
+		int miniAdvClass = get_property("miniAdvClass").to_int();
+		if(miniAdvClass == 1 || (miniAdvClass == 2 && my_level() >= 5) || (miniAdvClass == 3 && my_level() >= 15) ||
+		(miniAdvClass == 4 && my_level() >= 5) || (miniAdvClass == 4 && my_level() >= 10))
```

Comment:
`(miniAdvClass == 4 && my_level() >= 5) || (miniAdvClass == 4 && my_level() >= 10)`
This looks like an error. you are checking class 4 both times. either level 5+ or 10+
either the second one should have been a different class, or it is wholly redundant

Reply:
error indeed

also added slightly more accurate return for Reagnimated Gnome